### ISSUE TITLE
Uncaught AttributeError in sendchat.py line 30

### ISSUE THREAD

**Issue by bjornslib** [paul-gauthier/aider#1629]
Aider version: 0.56.0
Python version: 3.10.10
Platform: Linux-5.15.0-1070-aws-x86_64-with-glibc2.31
Python implementation: CPython
Virtual environment: No
OS: Linux 5.15.0-1070-aws (64bit)
Git version: git version 2.42.0

An uncaught exception occurred:

```
Traceback (most recent call last):
  File "base_coder.py", line 1124, in send_message
    continue
  File "base_coder.py", line 1408, in send
    self.keyboard_interrupt()
  File "base_coder.py", line 1490, in show_send_output_stream
    pass
aider.coders.base_coder.FinishReasonLength

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "aider", line 8, in <module>
    sys.exit(main())
  File "main.py", line 698, in main
    return 1
  File "base_coder.py", line 735, in run
    return self.io.get_input(
  File "base_coder.py", line 778, in run_one
    def check_for_urls(self, inp):
  File "base_coder.py", line 1126, in send_message
    interrupted = True
  File "sendchat.py", line 30, in retry_exceptions
    litellm.llms.anthropic.chat.AnthropicError,
AttributeError: module 'litellm.llms.anthropic' has no attribute 'AnthropicError'

```

**Comment by fry69** [comment]
Thank you for filing this issue.

The fix is in the latest version (`0.57.1`), please update aider with
```
aider --upgrade
```

**Comment by paul-gauthier** [comment]
Thanks for trying aider and filing this issue.

This looks like a duplicate of #1278. Please see the comments there for more information, and feel free to continue the discussion within that issue.

I'm going to close this issue for now. But please let me know if you think this is actually a distinct issue and I will reopen this issue.
